new-site/src/docs/getting-started/hds-2.0/migrating-to-2.0.mdx

```diff
@@ -0,0 +1,62 @@
+---
+slug: "/getting-started/hds-2.0/migrating-to-2.0"
+title: "Migrating to HDS 2.0"
+nav_title: "Migrating to HDS 2.0"
+---
+
+import { Notification } from "hds-react";
+
+import LeadParagraph from '../../../components/LeadParagraph';
+
+# How to migrate to HDS 2.0?
+
+<LeadParagraph>
+  Here you will find some useful information to make your transition to HDS 2.0 as smooth as possible.
+</LeadParagraph>
+
+Reading this article will help you understand what you should consider and take into account when you migrate your project to the HDS 2.0. If you are looking for a general overview of changes in version 2.0, please refer to our [What is HDS 2.0 page](/getting-started/hds-2.0/what-is-hds-2.0).
+
+## Developers
+
+<Notification label="A summary for developers" className="siteNotification">
+  <ol>
+    <li>Check components that have had their API (theme variables or default property values changed)</li>
+    <li>Check your tests to see if any of the visual changes have caused them to break</li>
+  </ol>
+</Notification>
+
+Before updating, we recommend to take a look at the [What is HDS 2.0 page](/getting-started/hds-2.0/what-is-hds-2.0) and <Link size="M" href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v2.0.0" external>the official 2.0 changelog</Link>. Pay extra attention to the "Breaking changes" section of the changelog. This helps you to get an understanding of what has been changed and what kind of effects updating may have in the implementation.
+
+Start the migration process by updating your project's libraries to the newest HDS version. After this, keep reading below to get familiar with all major changes and how you may need to react to those.
+
+### Changes to component APIs
+
+There are only a few breaking API changes in the HDS 2.0. 
+
+First, we have removed two (2) theme properties. These variables are not available for customisation anymore.
+
+| Component         | Removed theme variable        |
+| ----------------- | ----------------------------- |
+| [Accordion]()     | `--button-border-color-hover` |
```

Comment:
Should we have link to component documentation here?

Reply:
- [ ] Added missing 2.0 guide links